bin/cli.js

```diff
@@ -121,8 +118,10 @@ function dependencyFilter() {
 			const dir = path.dirname(relPath) + '/';
 			const file = path.basename(relPath);
 
-			spinner.text = chalk.grey(dir) + chalk.cyan(file);
-			spinner.render();
+			if (program.spinner) {
```

Comment:
Does it fail without this check?

Reply:
Without this check, the spinner will still render and print its text, which will still look like a mess in CI. This code is actually the source of unwanted text in CI. 

The spinner's enable option only affects animation of its image; it does not prevent the spinner from rendering when it is called to do so. However, the spinner already checks for CI and is implemented to disable animation if running in CI. I force enable=false here as well because this spinner option is independent of CI detection.